python-package/xgboost/plotting.py

```diff
@@ -0,0 +1,239 @@
+# coding: utf-8
+# pylint: disable=too-many-locals, too-many-arguments, invalid-name,
+# pylint: disable=too-many-branches
+"""Plotting Library."""
+from __future__ import absolute_import
+
+import re
+import numpy as np
+from .core import Booster
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+
+def plot_importance(booster, ax=None, cmap='cool', height=0.2, xlim=None,
+                    title='Feature importance',
+                    xlabel='F score', ylabel='Features',
+                    grid=True, **kwargs):
+
+    """Plot importance based on fitted trees.
+
+    Parameters
+    ----------
+    booster : Booster or dict
+        Booster instance, or dict taken by Booster.get_fscore()
+    ax : matplotlib Axes, default None
```

Comment:
Thanks. Fix keywords as below:

- Use ``num_tree`` rather than ``n``. 
- Remove ``cmap``. Because performing clustering seems to be difficult from current python impl.
- Still use ``ax``, because ``ax`` is popular in packages uses matplotlib. See: 
    - http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.plot.html
    - http://stanford.edu/~mwaskom/software/seaborn/generated/seaborn.violinplot.html